### ISSUE TITLE
createList contains duplicate sequence values

### ISSUE THREAD

**Issue by johannesstricker** [thoughtbot/fishery#73]
Hi :wave: I'm using version 1.4.0 and I think I've found an issue with the `createList` method. Using it sometimes results in duplicate sequence values. This is my test code:

```
  describe('findActiveUsers', () => {
    describe('as admin', () => {
      beforeEach(async () => {
        currentUser = await userFactory.roleAdmin().create();
      });

      it('returns all active users', async () => {
        const activeUsers = await userFactory.createList(5);
        console.log(currentUser.id);
        console.log(activeUsers.map((u) => u.id));
     // ...
```

This results in the following log output:
```
1
[2, 3, 2, 5, 4]
```

As you can see there is a duplicate id. I'm using [TypeORM](https://github.com/typeorm/typeorm) and the user entity has an `id` attribute that is a [@PrimaryGeneratedColumn()](https://github.com/typeorm/typeorm/blob/master/docs/entities.md#primary-columns). Could that be an issue? I think it shouldn't be, because I'm assigning the factory sequence to the user's `id` attribute:
```
class UserFactory extends Factory<User> {/* some traits */}

export default UserFactory.define(({ sequence, onCreate }) => {
  onCreate(async (user) => {
    return getRepository(User).save(user);
  });

  return getRepository(User).create({
    id: sequence,
    // more attributes
  });
});
```

🤷‍♂️

**Comment by stevehanson** [comment]
@johannesstricker I believe this might have been fixed in 2.0.0. Could you please upgrade to 2.0.0 or later and let me know if this addressed your issue? Alternatively, if you can create a complete reproducible code sample that demonstrates the problem, I could try it out to confirm.

**Comment by stevehanson** [comment]
Closing due to inactivity.
